_posts/2019-01-17-multi-store-custom-telemetry-with-shared-data.md

```diff
@@ -0,0 +1,188 @@
+---
+permalink: "/{{ year }}/{{ month }}/{{ day }}/multi-store-custom-telemetry-with-shared-data"
+title: "multi-store - Custom Telemetry with shared data"
+published_date: "2019-01-18 12:00:00 +0100"
+layout: post.liquid
+data:
+  route: blog
+---
+
+Last year I implemented a new feature for Firefox Telemetry that changes how we can collect and analyze data with different requirements in regard to user privacy & frequency.
+This post will shine some light on the (rather simple) implementation and usage.
+
+### Intro: What is Firefox Telemetry?
+
+In order to understand how Firefox performs in the wild, it can collect a bunch of performance metrics and other information.
```

Comment:
Could add a nice link here that clearly introduces people to our Firefox data practices?
A good blog post or wiki article or similar. Maybe Chris has a good link handy?